SonarProjectConfig.xml

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Filename itself is also to be reviewed.
`Sonar` - because additional files are passed to all analyzers
`Project` - because it's project-level context
`Config` - because it configures behavior of the analyzer on input side

Reply:
where will this be stored under `C:\Projects\1h with Hacker\AwesomeBankWeb\.sonarqube\out\0`? if yes, it makes sense.

however, there will also be `ProjectInfo.xml` which is a "SonarProjectConfig" like file , too

could you name it `SonarAnalyzersProjectConfig`, to make it explicit that only analyzers will use it?

Reply:
or even just AnalyzersProjectConfig, because everything under the `.sonarqube` folder is "Sonar"

Reply:
No, this will be under `.sonarqube\conf\0` folder.
All additional files are passed to all analyzers => we need "Sonar" in the filename.